tests/Microsoft.Bot.Builder.TestBot/BotBuilderTestBot.bot

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "BotBuilderTestBot",
+    "description": "",
+    "secretKey": "",
+    "services": [
+        {
+            "appId": "",
+            "id": "98fd9150-a278-11e8-90fe-f92f88ed2f78",
+            "type": "endpoint",
+            "appPassword": "",
+            "endpoint": "http://localhost:51135/api/messages",
```

Comment:
This port I think changes based on the developer... I'm not familiar with debugging in VS, but I don't believe every spun up bot will use this port.

Reply:
@stevengum @johnataylor  same applies to the id tag, but I don't think that has any material impact on a local emulator, but might impact actual ABS deployment.  @vishwacsena what is the recommended .bot file setup for templates and examples?